doc/libfuzzer.md

```diff
@@ -0,0 +1,57 @@
+# libfuzzer
+
+## setup 
+
+### Requirements
+libFuzzer needs LLVM sanitizer support, 
+so this only works on x86-64 Linux, 
+x86-64 macOS and Apple-Silicon (aarch64) macOS for now. 
+This also needs a nightly compiler since 
+it uses some unstable command-line flags. 
+You'll also need a C++ compiler with C++11 support.
+
+
+
+### Installing
+
+rust-version|libfuzzer|td-shim|result
+:-:|:-:|:-:|:-:
+nightly-2020-02-08|0.11.0|staging|no (td-shim fails to compile in this version)
+nightly-2021-08-20|0.10.2|staging|yes
+nightly-2020-02-08|0.11.0|staging|no (libfuzzer failed to start)
+
+```
+cargo install cargo-fuzz --version 0.10.2
+# coverage
+rustup component add llvm-tools-preview rust-src
+cargo install grcov
+```
+### Provide starting inputs
+
+Use RAMdisks for input since, we don't want to destroy harddrives
+
+```
+$sudo mount -t tmpfs -o size=1024M tmpfs fuzz/corpus
+```
+
+### Example
+
+```
+cd td-loader
+cargo fuzz list 
+cp fuzz/seeds/pe_send/td-shim.efi fuzz/corpus/pe
+cargo fuzz run pe
+# If there are no errors it will keep running, you can use Ctrl + c to exit
+```
+
+### View coverage
+
+```
+cargo fuzz coverage pe
+grcov . -s . --binary-path ./fuzz/target/x86_64-unknown-linux-gnu/release/ \
```

Comment:
I'd consider removing the backslash and letting GitHub auto render scroll bars.

Reply:
Done